### PR TITLE
#883 issue, Roles Display Name

### PR DESCRIPTION
**Pull request by shivasheeshyadav**
Display Names of Roles updated, Issue #883

### CODE REVIEW

File: server/constants.py [hunk 1]
```diff
@@ -5,6 +5,13 @@
 STAFF_ROLE = 'staff'
 INSTRUCTOR_ROLE = 'instructor'
 LAB_ASSISTANT_ROLE = 'lab assistant'
+ROLE_DISPLAY_NAMES = {
+    'student' : 'Student',
+    'grader' : 'Reader',
+    STAFF_ROLE : 'Teaching Assistant',
```
Comment (knrafto) [inline server/constants.py]: There should not be a space before the colon. Also, what about the rest of the repeated string literals?

File: server/forms.py [hunk 2]
```diff
@@ -168,7 +168,7 @@ class EnrollmentForm(BaseForm):
     section = StringField('Section',
                           validators=[validators.optional()])
     role = SelectField('Role', default=STUDENT_ROLE,
-                       choices=[(r, r.title()) for r in VALID_ROLES])
+			choices=ROLE_DISPLAY_NAMES.items())
```
Comment (knrafto) [inline server/forms.py]: Please indent using spaces, not tabs

File: server/forms.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
 from server.models import Assignment, Course
 from server.constants import (VALID_ROLES, GRADE_TAGS, COURSE_ENDPOINT_FORMAT,
                               TIMEZONE, STUDENT_ROLE, ASSIGNMENT_ENDPOINT_FORMAT,
-                              COMMON_LANGUAGES)
+                              COMMON_LANGUAGES,ROLE_DISPLAY_NAMES)
```
Comment (knrafto) [inline server/forms.py]: There should be a space after the comma (but not before) here

**Comment by shivasheeshyadav** [comment]
@knrafto Done with the requested changes.

**Comment by Sumukh** [comment]
Looks fine to me. 

For future improvement - it should also be changed here: 
https://github.com/Cal-CS-61A-Staff/ok/blob/master/server/controllers/admin.py#L712

image

**Pull request merged**